07-regression1.Rmd

```diff
@@ -123,7 +135,6 @@ This function takes two arguments:
 2. `size` (the number of observations/rows to be randomly selected/sampled)
 
 ```{r 07-sacramento}
-set.seed(1234)
```

Comment:
Why are we not showing the seed here? `sample_n` is taking a random sample, and if someone wanted to follow our example, they would need this seed.

Reply:
dealing with seeds is in a formatting pass issue